docs/data-sources/scalr_policy_group.md

```diff
@@ -43,6 +43,7 @@ The `vcs_repo` object contains:
 * `identifier` - A reference to the VCS repository in the format `:org/:repo`, it stands for the organization and repository.
 * `branch` - A branch of a repository the policy group is associated with.
 * `path` - A subdirectory of a VCS repository where OPA policies are stored.
+* `ingress_submodules` - Designates whether to clone git submodules of the VCS repository
```

Comment:
Let's consolidate descriptions for `ingress_submodules` attribute, right now they differ in different resources.

Reply:
Updated.